src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -2832,6 +2844,202 @@ ImmutableArray<IArgumentOperation> makeArguments(Lazy<ImmutableArray<IArgumentOp
+                                }
+                            }
+                            else if (!rightIsNullable && rightOperand.Type != null)
+                            {
+                                // PROTOTYPE(dataflow): Consider using similar conversion for TryMakeNullable value and using the helper here
+                                rightOperand = new ConversionOperation(rightOperand, ConvertibleConversion.Instance, isTryCast: false, isChecked: false,
+                                                                       semanticModel: null, rightOperand.Syntax, operation.Value.Type,
+                                                                       constantValue: default, isImplicit: true);
+                            }
+                        }
+
+                        condition = new BinaryOperatorExpression(BinaryOperatorKind.Equals,
+                                                                 leftOperand,
+                                                                 rightOperand,
+                                                                 isLifted,
+                                                                 isChecked: false, 
+                                                                 isCompareText: false,
+                                                                 operatorMethod: null,
+                                                                 semanticModel: null,
+                                                                 singleValueClause.Value.Syntax,
+                                                                 ((Operation)operation).SemanticModel.Compilation.GetSpecialType(SpecialType.System_Boolean), 
+                                                                 constantValue: default, 
+                                                                 isImplicit: true);
+
+                        condition = Operation.SetParentOperation(condition, null);
+                        LinkBlocks(CurrentBasicBlock, (condition, JumpIfTrue: false, RegularBranch(nextCase)));
+                        AppendNewBlock(labeled);
+                        _currentBasicBlock = null;
+                        break;
+
+                    case CaseKind.Default:
+                        var defaultClause = (IDefaultCaseClauseOperation)caseClause;
+                        if (defaultBody == null)
+                        {
+                            defaultBody = labeled;
+                        }
+
+                        // 'default' clause is never entered from the top, we'll jump back to it after all 
+                        // sections are processed.
+                        LinkBlocks(CurrentBasicBlock, nextCase);
+                        AppendNewBlock(labeled);
+                        _currentBasicBlock = null;
+
+                        break;
+                    default:
+                        // PROTOTYPE(dataflow): TODO.
```

Comment:
> TODO what? It would be good to have a list

This is coming in my next PRs

---
In reply to: [184241555](https://github.com/dotnet/roslyn/pull/26395#discussion_r184241555) [](ancestors = 184241555)